### DISCUSSION TITLE
[Kavita] Sort by Random

### DISCUSSION THREAD

**Discussion by lyingloverr** [Kareadita/Kavita#2645] [Ideas]
### Idea Description

I think it would be really nice to be able to **_sort by random_** for a little charm. You can take a peak at my **Jump In Early** and **Short Reads** filter in the `#awesome-smart-filters` channel, and I think those filters, for example, would benefit from a sort by random. Being sorted by random, the way I would expect it to work, is on every login, or page refresh, or even a 24 hour period, the sorting order would change. 

This is mostly to benefit the use case of including an aforementioned smart filter on your dashboard.

Open to suggestions of course!

### Duration of Using Kavita

2 Months

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
This has been implemented:
https://github.com/Kareadita/Kavita/pull/2917
